js/bowling.js

```diff
@@ -0,0 +1,43 @@
+var Bowling = function() {
+  this.pins = 10;
+  this.frameNumber = 1;
+  this.rollNumber = 1;
+  this.strikes = 0;
+  this.spares = 0;
+  this.currentScore = 0;
+};
+
+Bowling.prototype.pinsHit = function(number) {
+  if(number === 10)
+    {this.strikes += 1;
+    this.rollNumber += 1;
+    this.scoreTotal();
+    this.frameNumber += 1;}
+  else{this.pins = (this.pins -= number);
+    this.rollSpare();
+    this.nextFrame();}
+};
+
+Bowling.prototype.rollSpare = function() {
+  if(this.pins === 0) {this.spares += 1;}
+};
+
+Bowling.prototype.nextFrame = function() {
+  if(this.rollNumber === 1) {this.rollNumber += 1;}
+  else if(this.rollNumber === 2)
+    {this.rollNumber -= 1;
+    this.scoreTotal();
+    this.frameTotal();
+    this.pins = 10;}
+};
```

Comment:
What executes if neither of these conditions is true? Or did you mean to use else?